.github/workflows/pr-checks.yml

```
@@ -40,16 +40,13 @@ jobs:
         changed-files: ${{ steps.setup.outputs.changed_files }}
         files: '.eslintrc.js,package.json,.eslintignore'
 
-    - name: Full ESLint
-      if: steps.eslint-reset.outputs.filtered != '[]'
-      run: yarn eslint .
-
-    - name: Limited ESLint
-      if: steps.eslint-reset.outputs.filtered == '[]' && steps.js-files.outputs.filtered != '[]'
-      uses: ./.github/actions/json-args
+    - name: Eslint
+      uses: ./.github/actions/full-or-limited
       with:
-        list: ${{ steps.js-files.outputs.filtered }}
-        run: yarn eslint {}
+        full-trigger: ${{ steps.jest-reset.outputs.filtered }}
```

I think we want the `eslint-reset` output, right? :) 

```suggestion
        full-trigger: ${{ steps.eslint-reset.outputs.filtered }}
```